user_manual/changes/S912-013.yaml

```diff
@@ -0,0 +1,10 @@
+type: new-feature
+title: Add P_Unit_Dependencies property
```

Comment:
You documented the addition of this property but not `withed_units` (and `imported_units` if you make it public too)